EazyE2E/Process/EzProcess.cs

```diff
@@ -1,42 +1,69 @@
 ﻿using System;
 using System.Diagnostics;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading;
 
 namespace EazyE2E.Process
 {
-    public class EzProcess
+    public class EzProcess : IDisposable
```

Comment:
Implement IDisposable to allow us to use EzProcess in a using statement.  This lets consumers of the library know there are resources to be cleaned up.  Namely, the process that is being started.